specification/advisor/resource-manager/Microsoft.Advisor/stable/2023-01-01/advisor.json

```diff
@@ -140,6 +140,68 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/configurations/v2/{configurationName}": {
```

Comment:
>v2 [](http://example.com/codeflow?start=80&length=2)

Can you add some context on this API? What is v2 and this only has PUT, or GET or DELETE or PATCH support?

Reply:
So we needed a new route path because the return object includes a new parameter. Basically I added the v2 to signify that this is the updated configurations route. and yes the supported operations did not change from the first configurations route.

Reply:
You can't do this because it breaks the {typeCollection}/{itemName} path convention... in fact the problem "because the return object includes a new parameter." sounds like it should be solved with api versioning, to me.